doc/Development_Documentation/20_Extending_Pimcore/13_Bundle_Developers_Guide/07_Adding_Asset_Types.md

```diff
@@ -0,0 +1,50 @@
+# Adding Asset Types
+
+This feature allows users to add their own custom asset types.
+To register new custom asset types, you must follow these steps:
+
+
+## 1) Create the PHP asset class
+
+The asset **must** extend `Pimcore\Model\Asset`. Let´s create a for a InDesign file (the namespace does not matter
+but it's best practice to put your assets into a `Model\Assets` sub-namespace):
+
+For examples have a look at the Pimcore core asset types at
+[github](https://github.com/pimcore/pimcore/tree/11.x/models/Asset).
+
+```php
+<?php
+// src/Model/Assets/InDesign.php
+
+namespace App\Model\Asset;
+
+class InDesign extends \Pimcore\Model\Asset
+{
+    protected string $type = 'indesign';
+}
+```
+
+## 2) Create JavaScript class for the asset view editor:
+
+It needs to extend `pimcore.asset.asset`, be located in namespace `pimcore.asset` and named after the
+`$type` property of the corresponding PHP class.
+
+For examples have a look at the Pimcore core asset types at
+[github](https://github.com/pimcore/pimcore/tree/11.x/bundles/AdminBundle/public/js/pimcore/asset)
+
+## 2) Register the asset on the asset type map
+
+Next we need to update `pimcore.assets.type_definitions.map` configuration to include our asset. This can be done in any config
```

Comment:
```suggestion
Next we need to update the `pimcore.assets.type_definitions.map` configuration to include our asset. This can be done in any config
```